Additional Info Tabs Added

FeyFaelis opened the pull request:
I've added a good amount of them now. Smile.

FeyFaelis committed:
Added Markdown Files.

Added all the required files, Empty right now.

FeyFaelis committed:
Marketplace Disc

Renamed files because I am stupid and updated Marketplace with basic information.

FeyFaelis committed:
G U I D E S

Finished up some more guides. I went crazy with Settings.

FeyFaelis committed:
Forgot one.

whoops lmfao. Added a small disc to Session stats

IceFreez3r commented on Marketplace.md:
```diff
@@ -0,0 +1,7 @@
+The Market Place is where you'll go in order to sell and buy other items from players. You can get here by clicking Marketplace on the side or clicking on your gold bag above your stockpile. You can see all the various items within the game here that other players are selling. When you click on any item, it'll open a different window that will show the different listings by other players. By defa ...
```
> By default it is *~~listed~~ordered* by price, lowest to *high~~i~~est*.

IceFreez3r commented on PlatShop.md:
```diff
@@ -0,0 +1,7 @@
+Platinum (Plat) can be purchased with real life money in order to obtain global player buffs, quality of life tokens, and cosmentic items. 
```
cosmentic

IceFreez3r commented on PlatShop.md:
```diff
@@ -0,0 +1,7 @@
+Platinum (Plat) can be purchased with real life money in order to obtain global player buffs, quality of life tokens, and cosmentic items. 
+<br />
+Global player buffs will apply to the entire server for others to enjoy, as well as a special thank you in general chat when purchased. You can opt out of displaying your name by clicking "Anonymous Purchase". 20 Plat is equal to one minute of buff time. Buffs start out at level 2, but can upgrade to level 3 and 4 once certain time threshholds are met.
```
Level 3 is 1 day. Level 4 is 1 week. Once the threshold is reached, the buff stays at that level including later extensions by different players or yourself

IceFreez3r commented on Settings.md:
```diff
@@ -0,0 +1,52 @@
+Show Farming Notifications: If you want to see farming harvests or totem actions.
+Show Augmenting Results: If you want to see Augment Success or Failures.
+Show Action Notifications: If you want to see messages relating to your current action.
+Show Combat Drops: If you want to see drops from combat.
+Show Combat Timers/Warnings: If you want to see Timers and Warnings from Combat. (like enrage)
+Show Combat Experience: If you wish to see experienced earned from monsters in combat on kill.
+Show Rare Drops: If you want to see any Rare drops you've received.
+Show Flavor Messages: If you want to see special messages from certain zones and actions.
+Show Enchantment Procs: If you want to see procs from your Enchantments (Like Inferno).
+Show Shard Messages: If you want to see when you earned a shard from skilling.
+<b> Sound </b>
+Sound on Mention: If a sound plays when your name is mentioned in chat.
+Sound on Whisper: If a sound plays when a player whispers you.
+Sound on Group Invite: If a sound plays when an invite is received.
+Volume: How loud you want it to be.
+<b> Combat </b>
+Auto Eat: What threshhold of percent HP you wish to automatically eat food to heal yourself at.
+Max Difficulty: What you wish to set your Global Max Difficulty at.
+Starting Difficulty: What you wish to set your Global Starting Difficulty at.
+Force Full Inventory: If you want your food inventory to be full before it starts combat. Will not let you start combat if its not full if active.
+Show Inventory while Grouped: If you want to see your inventory while grouped with others.
+Allow Invites from: Select who is allowed to invite you.
+Automatic Priority Rotation: If you want the game to automatically set your combat skill rotation.
+Force Valid Merc Loadout: If you want the game to make sure your merc's are properly equipped before combat. Will not start combat if invalid if active.
+Dungeon Misclick Safety: If you want to prevent accidently leaving dungeons.
+Ignore Dungeon Update Safety: ?
+Target Threat Bars: Your combat targeting priority. Does not fully work as intended.
+<b> Tooltips </b>
+Modern Style: A more modern style of tooltip for your items. Turn off to use oldschool Tooltips
+Compact Tooltips: Weither to hide overly bloated information. Will reduce tooltip size.
+Equipped Comparison: Weither to show your currently equipped items when you hover over an item that goes into that equip slot.
+<b> Privacy </b>
+Anonymous: If you want your name to be displayed or not in Community Data Releases.
+<b> Misc </b>
+Auto Vault Like Items: If you want items you obtained to automatically be put in the vault if they are already in your vault.
+Auto Tacklebox Like Items: If you want bait to automatically be put in your tacklebox if they are already in there.
+Use Legacy Sorting: If you want to sort items the way they used to be sorted. (This will ignore other settings)
+Sort by Value before ID: If you want to sort items by value instead of their item ID's
+High Performance Mode: Will disable some animations and graphics to prioritize Performance.
+Enable Pinned Tabs: If you want to pin tabs for quicker access.
```
Would prefer if we have a more general description here. Otherwise it will be outdated in a week :D

FeyFaelis replied:
Fair

FeyFaelis replied:
I'll move the setting info to a separate note file just in case.

IceFreez3r commented on Shrine.md:
```diff
@@ -0,0 +1,6 @@
+In order to make an offering, input the amount of gold you wish to offer in the text box below, then click "OFFER" to confirm. 
+<br />
+<b> Buffs that can appear </b>
+(List of buffs that can appear within the shrine so players can stop asking.)
```
Likely to be forgotten aswell, but at least the buffs here don't change a lot

FeyFaelis replied:
I'll remove it then, just some food for thought.

IceFreez3r commented on Marketplace.md:
```diff
@@ -0,0 +1,7 @@
+The Market Place is where you'll go in order to sell and buy other items from players. You can get here by clicking Marketplace on the side or clicking on your gold bag above your stockpile. You can see all the various items within the game here that other players are selling. When you click on any item, it'll open a different window that will show the different listings by other players. By defa ...
+<br />
+<b> Selling </b>
+In order to sell items on the market, you can click the "Sell" button also in the top left on the main screen. Here you will see all the items you can sell, as well as items you already have listed on the market if any. When you click on an item, it will bring up a screen asking you to select the amount you wish to sell of that item, and for how much. Once you are satisfied with your listing, cli ...
```
Somethink similar to this
Tip: You can use K,M,B in the price input. E.g. 2.5M for 2,500,000
would be nice.

FeyFaelis committed:
Comment Updates

Changed and Updated based from comments after the review. Moved Settings into SettingsTooltips for future use maybe.

FeyFaelis commented:
Fixed and added based upon comments made.

FeyFaelis committed:
Guide Updates

Updates to the guides based on community feedback

FeyFaelis commented:
Updated guides based on community feedback

FeyFaelis committed:
Marketplace fix

I swear I added this with the push
